casbin/.clang-format

```diff
@@ -0,0 +1,25 @@
+BasedOnStyle: Google
+PointerAlignment: Right
+AccessModifierOffset: -4
+IndentWidth: 4
+MaxEmptyLinesToKeep: 1
+BreakBeforeBraces: Attach
+AllowShortFunctionsOnASingleLine: true
+AllowShortIfStatementsOnASingleLine: true
+AlignAfterOpenBracket: true
+IndentCaseLabels: true
+ObjCBlockIndentWidth: 4
+ObjCSpaceAfterProperty: true
+ColumnLimit: 0
+AlignTrailingComments: true
+SpaceAfterCStyleCast: false
+AlignOperands: true
+SpacesInSquareBrackets: false
+AlignConsecutiveDeclarations: false
+SpacesInContainerLiterals: false
+BreakConstructorInitializersBeforeComma: true
+AllowAllParametersOfDeclarationOnNextLine: true
+ContinuationIndentWidth: 4
+TabWidth: 4
+SpaceBeforeAssignmentOperators: true
+SpacesBeforeTrailingComments: 1
```

Comment:
These styles are too strict, I think you can choose google as base style then add several your own styles.